src/hooks/useMe.ts

```diff
@@ -21,11 +20,12 @@ interface MeResponse {
 }
 
 export function useMe() {
+  const { authResult } = useAuth();
   const { accountHeaders } = useActiveAccount();
   const { httpClient } = useHttpClient();
 
   const useMeQuery = useQuery(
-    ['fhir/dstu3/$me'],
+    ['fhir/dstu3/$me', authResult?.accessToken],
```

Comment:
This doesn't seem quite right. This would fire even when the access token is refreshed.

Reply:
Maybe we should consider adding a sessionId to `useAuth` to handle this for us. It could just be a unique string (timestamp?) that changes whenever a user logs in for the first time after being logged out.

Reply:
Updated to bust cache when the inviteId changes which should work nicely.